### ISSUE TITLE
Ban bug

### ISSUE THREAD

**Issue by TrueReassembly** [Minehut/Meta#332]
### Bug report

**Checklist**
- [x] I have read and agree to the README (found on https://github.com/Minehut/Meta)
- [x] I am NOT reporting a security vulnerability
- [x] I have searched the Issue Tracker to make sure this is not a duplicate issue
- [x] This issue does not contain any sensitive information

**Platform**
 Ingame

**Describe the bug**
 If you ban someone (Only tested with skript) and the reason contains "minehut.com" it will disconnect you from the entire network.

**To reproduce**
Steps to reproduce the behaviour:
Steps said above

**Expected behaviour**
 You would expect it to just send you to the lobby

**Device Information**

- Minecraft client (and version): 1.16.3
- Web browser (and version): Not needed
- Operating system (and version): Not needed
- Minehut server name: Not needed, works globally

**Screenshots**

**Additional context**

---
**Try to reproduce this bug and let us know in the comments if you can reproduce it or not. Make sure to include your relevant device information!**

**Comment by LukasmanMHdude** [comment]
Haven't people been asking for this for a while? This is a feature, not a bug.

**Comment by Blocksnmore** [comment]
It is a bug, typically it sends you to the lobby but aparently if the reason contains minehut.com it kicks from the network

**Comment by Sniped** [comment]
Yeah, for any other kick/ban reason it would just send them back to the lobby. This is unexpected behavior, therefore it's a bug.

Can reproduce this bug on my server with vanilla punishment commands.

**Comment by mekb-turtle** [comment]
I'm guessing this is because when a Minehut staff member punishes you, the disconnect message contains Minehut.com
